src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
let's make UserService class-level variable to avoid creating this object each time you call login method

Reply:
Ok. I thought that since Java take care of automatically destruction of object (because it doesn't have explicitly defined destructors) - it would be right to create this object inside of method, just to renew users datarecords each time when method is called. Second reason - what will happen if between method calls to the User will be added new authentication records...